pkg/test/deployment/helm.go

```diff
@@ -107,7 +107,17 @@ func HelmTemplate(deploymentName, namespace, chartDir, valuesFile string, values
 		valuesFileString = fmt.Sprintf(" --values %s", valuesFile)
 	}
 
-	str, err := shell.Execute(
+	str, err := shell.Execute("helm init --client-only")
```

Comment:
@sdake this seems to have broken the some tests: #9405.

Running locally, it seems to return an empty string here instead of the attribute manifest (requested here: https://github.com/istio/istio/blob/master/pkg/test/deployment/attrmanifest.go#L30).  What is the right fix here?

FYI @ozevren

Reply:
@nmittler  -- I don't know this test code but looks like the fix is `err == nil` => `err != nil` here and L116 since those steps are prereqs for L120's `helm template ...` exec

Reply:
:) yeah I assumed that as well. Just wanted to pass it by @sdake for verification.

I have #9434 out for review.

Reply:
@nmittler  -- cool... fyi, @sdake is on PTO today through next week.  I'll try to help out on issues as a result of this change as well.